code/game/objects/items/cards_ids.dm

```diff
@@ -139,8 +142,8 @@
 /obj/item/card/id/Destroy()
 	if (registered_account)
 		registered_account.bank_cards -= src
-	if (my_store)
-		QDEL_NULL(my_store)
+//	if (my_store)
```

Comment:
Mark your edits with /// LETHAL EDIT REMOVAL on all da lines you single remove like this, or in big group comments so that way if i need to find why something exploded i can control f LETHAL EDIT and find everything we fucked up